actionpack/test/dispatch/ssl_test.rb

```diff
@@ -39,6 +39,18 @@ def assert_redirected(redirect: {}, deprecated_host: nil, deprecated_port: nil,
     assert_equal redirect[:body].join, @response.body
   end
 
+  def assert_post_redirected(redirect: {}, deprecated_host: nil, deprecated_port: nil,
+    from: 'http://a/b?c=d', to: from.sub('http', 'https'))
+
+    self.app = build_app ssl_options: { redirect: redirect,
+      host: deprecated_host, port: deprecated_port
```

Comment:
Let's just remove the deprecated stuff from here, they aren't long for this world.